accord-core/src/test/java/accord/burn/BurnTest.java

```diff
@@ -243,6 +283,12 @@ static void burn(RandomSource random, TopologyFactory topologyFactory, List<Id>
                 if (i == requests.length - 1)
                     onSubmitted.get().run();
             }
+            if (packet.message instanceof Reply.FailureReply)
```

Comment:
hit a NPE in the `List*` code which hit this case... so assuming the `List*` is bug free we should never hit this, but rather than hitting a class cast felt that the original exception was best so adding directly to the failures was better